### PR TITLE
Redesign heading and add buttons

### PR DESCRIPTION
**Pull request by florisvdg**
4 changes:
- The two logos have been replaced with an image which includes a `+` and a headline
- Two buttons have been added, linking externally to the integration page and to the guide
- The project title is not italic anymore
- The description copy is updated

### CODE REVIEW

**Review by mackenbach — APPROVED** [review]

**Comment by mackenbach** [comment]
One thing: how does this affect the display on the Orb 'store'?

**Comment by florisvdg** [comment]
> One thing: how does this affect the display on the Orb 'store'?

It doesn't. There is no relation between the orb registry and the GitHub README.

File: README.md [hunk 1]
```diff
@@ -1,23 +1,24 @@
 <p align="center">
-  <a href="https://circleci.com">
-    <img src="https://upload.wikimedia.org/wikipedia/commons/8/82/Circleci-icon-logo.svg" alt="CircleCI" width="110">
-  </a>
-  <img width="50px"/>
-  <a href="https://secrethub.io">
-    <img src="https://secrethub.io/img/secrethub-logo-shield.svg" alt="SecretHub" width="96">
+  <img src="https://secrethub.io/img/integrations/circleci/github-banner.png?v1" alt="CircleCI + SecretHub" width="390">
   </a>
```
Comment (SimonBarendse) [inline README.md:3]: Missed a `</a>`, I'll open a new PR.

Comment (SimonBarendse) [reply README.md:3]: https://github.com/secrethub/secrethub-circleci-orb/pull/30

**Pull request merged**